src/App.js

```diff
@@ -3,17 +3,18 @@ import styled from 'styled-components';
 
 import ChattingScreen from './components/ChattingScreen';
 
-export default function App() {
-	return (
-		<Wrapper>
-			<ChattingScreen></ChattingScreen>
-		</Wrapper>
-	);
-}
+const App = () => {
+    return (
+        <Wrapper>
+            <ChattingScreen></ChattingScreen>
```

Comment:
내용이 없다면  
```suggestion
            <ChattingScreen/>
```
이렇게 하셔도 좋습니다

Reply:
아아! 넵넵 하나 알아갑니다!